src/app/components/feed/FeedContent.js

```diff
@@ -1,61 +1,78 @@
     <div className="typography-subtitle2">
       <FormattedMessage
         id="saveFeed.feedContentTitle"
-        defaultMessage="Feed content"
+        defaultMessage="Feed content from this workspace"
         description="Title of section where a list can be selected as the content of the feed"
       />
     </div>
+
     <div className="typography-body2">
-      <FormattedMessage
-        id="saveFeed.feedContentBlurb"
-        defaultMessage="Select a filtered list of fact-checks from your workspace to contribute to this shared feed. You will be able to update this list at any time."
-        description="Helper text for the feed content section"
-      />
-    </div>
-    <div className="typography-body2">
-      <FormattedHTMLMessage
-        id="saveFeed.feedContentBlurb2"
-        defaultMessage="<strong>Note:</strong> Your list must contain <strong>published fact-checks</strong> in order to be part of this shared feed."
-        description="Helper text for the feed content section"
-      />
+
+      {/* Feed is sharing fact-checks but not media/claim/requests */}
+      { dataPoints.includes(CheckFeedDataPoints.PUBLISHED_FACT_CHECKS) && !dataPoints.includes(CheckFeedDataPoints.MEDIA_CLAIM_REQUESTS) ?
+        <FormattedHTMLMessage
+          id="saveFeed.feedContentBlurbFactChecksOnly"
+          defaultMessage="Select a custom filtered list of <strong>published fact-checks</strong> from your workspace to contribute to this shared feed. You will be able to update this list at any time."
+          description="Helper text for the feed content section when the feed is sharing only fact-checks."
+        /> : null }
+
+      {/* Feed is sharing media/claim/requests but not fact-checks */}
+      { !dataPoints.includes(CheckFeedDataPoints.PUBLISHED_FACT_CHECKS) && dataPoints.includes(CheckFeedDataPoints.MEDIA_CLAIM_REQUESTS) ?
+        <FormattedHTMLMessage
+          id="saveFeed.feedContentBlurbMediaOnly"
+          defaultMessage="Select a custom filtered list of <strong>media, claim & requests</strong> from your workspace to contribute to this shared feed. You will be able to update this list at any time."
+          description="Helper text for the feed content section when the feed is sharing only media."
+        /> : null }
+
+      {/* Feed is sharing both fact-checks and media/claim/requests */}
+      { dataPoints.includes(CheckFeedDataPoints.PUBLISHED_FACT_CHECKS) && dataPoints.includes(CheckFeedDataPoints.MEDIA_CLAIM_REQUESTS) ?
+        <FormattedHTMLMessage
+          id="saveFeed.feedContentBlurbFactChecksAndMedia"
+          defaultMessage="Select a custom filtered list of <strong>published fact-checks or media, claim & requests</strong> from your workspace to contribute to this shared feed. You will be able to update this list at any time."
+          description="Helper text for the feed content section when the feed is sharing fact-checks and media."
+        /> : null }
+
```

Comment:
Is it possible that `dataPoints` would have none of these numbers in them? If so, should we catch that and display an error alert on `!dataPoints.includes(CheckFeedDataPoints.PUBLISHED_FACT_CHECKS) && !dataPoints.includes(CheckFeedDataPoints.MEDIA_CLAIM_REQUESTS)` ?

Reply:
Good question! No, there is a validation to be sure that there is at least one.